libsentrykube/tests/test_data/quickpatches/test-patch-override.yaml

```diff
@@ -0,0 +1,28 @@
+# Each patch file defines a patch for a single resource
+name: Scale Example
+schema:
+  type: object
+  properties:
+    replicas1:
+      type: integer
+      minimum: 1
+      maximum: 10
+    replicas2:
+      type: integer
+      minimum: 1
+      maximum: 10
+  required:
+    - replicas1
+    - replicas2
+  additionalProperties: false
+mappings:
+  # Only the keys here are valid resources which patches can be applied to
+  test-consumer-prod: consumer
+patches:
+  # Key-Values should follow jsonpatch format
+  - op: replace
+    path: /consumers
+    value: { "<resource>": { "replicas": <replicas1> } }
+  - op: replace
+    path: /consumers
+    value: { "<resource>": { "replicas": <replicas2> } }
```

Comment:
This will require a lot more test coverage. 
Please write a unit test that tests the  `patch_json` function in isolation.  Since that function is only operating on the state provided as an argument that is a perfect scenario for `@pytest.param`
Specifically test at least these cases:
- Invalid path
- `/` as path
- Multi level path `/asd/sdf/dfg/` etc.
- Case where the whole path is not present
- Case where part of the path is present 
- Update a literal value
- Apply a dictionary
- Apply a dictionary where another dictionary is present? Do you merge them ?
- Variable substitution 
- This path `/asd/sdf//////`